pandas/core/dtypes/dtypes.py

```diff
@@ -584,15 +592,15 @@ class DatetimeTZDtype(PandasExtensionDtype, ExtensionDtype):
     THIS IS NOT A REAL NUMPY DTYPE, but essentially a sub-class of
     np.datetime64[ns]
     """
-    type = Timestamp
-    kind = 'M'
+    type = Timestamp  # type: Type[Timestamp]
```

Comment:
What error was this throwing before? I think inference should be able to detect this